src/setup-jest.ts

```diff
@@ -2,12 +2,27 @@
 
 require('./reflect-metadata');
 
-require('zone.js/dist/zone.js');
-require('zone.js/dist/proxy.js');
-require('zone.js/dist/sync-test');
-require('zone.js/dist/async-test');
-require('zone.js/dist/fake-async-test');
-require('./zone-patch');
+let builtInJestSupportInZoneJs = false;
+try {
+  // If the user are using zone.js 0.11.1+
+  // all jest support logic are implemented inside zone.js
+  // we only need to load zone.js and zone.js/testing module
+  require('zone.js');
+  require('zone.js/testing');
+  if (!!jest['__zone_patch__']) {
+    builtInJestSupportInZoneJs = true;
+  }
+} catch (err) {}
+if (!builtInJestSupportInZoneJs) {
```

Comment:
e2e for `test-app-v10-zone-v11` failed due to this `builtInJestSupportInZoneJs` has `false` value

Reply:
Yeah, it seems the `jest` global instance are different in `zone-testing` and `setup-jest`. I will do a little more investigation.